3_Implementation/documentation/html/search/defines_3.js

```diff
@@ -0,0 +1,452 @@
+var searchData=
+[
+  ['test_5fabort_949',['TEST_ABORT',['../unity__internals_8h.html#a35ff53d9ad3c4ec9d611fdf04319a23b',1,'unity_internals.h']]],
+  ['test_5fassert_950',['TEST_ASSERT',['../unity_8h.html#ae0a72bc174ac97e90c4a711e5e52d540',1,'unity.h']]],
+  ['test_5fassert_5fbit_5fhigh_951',['TEST_ASSERT_BIT_HIGH',['../unity_8h.html#ab07290e7bac80e36e4f169f50da59632',1,'unity.h']]],
+  ['test_5fassert_5fbit_5fhigh_5fmessage_952',['TEST_ASSERT_BIT_HIGH_MESSAGE',['../unity_8h.html#a8cfe9a3f00f8cd17f076c9befb98821e',1,'unity.h']]],
+  ['test_5fassert_5fbit_5flow_953',['TEST_ASSERT_BIT_LOW',['../unity_8h.html#a539b80a45cda0f032f8ee011ab0b5d56',1,'unity.h']]],
+  ['test_5fassert_5fbit_5flow_5fmessage_954',['TEST_ASSERT_BIT_LOW_MESSAGE',['../unity_8h.html#af77907b390d72d2ab0c686fea1afef1f',1,'unity.h']]],
+  ['test_5fassert_5fbits_955',['TEST_ASSERT_BITS',['../unity_8h.html#ae76772524d720cfce161b2cb7cff8886',1,'unity.h']]],
+  ['test_5fassert_5fbits_5fhigh_956',['TEST_ASSERT_BITS_HIGH',['../unity_8h.html#a72097c41ab00631674be5ff2053c475b',1,'unity.h']]],
+  ['test_5fassert_5fbits_5fhigh_5fmessage_957',['TEST_ASSERT_BITS_HIGH_MESSAGE',['../unity_8h.html#a7e8a15745cc2b5c493231edfad158a8f',1,'unity.h']]],
+  ['test_5fassert_5fbits_5flow_958',['TEST_ASSERT_BITS_LOW',['../unity_8h.html#a10e2230d1ebe3fb1f00c5a81afe19b5c',1,'unity.h']]],
+  ['test_5fassert_5fbits_5flow_5fmessage_959',['TEST_ASSERT_BITS_LOW_MESSAGE',['../unity_8h.html#a58a1013db321e096d146bf7a90968f84',1,'unity.h']]],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/2015pushkar/C-mini-Project-303459-LTTS/pullRequest?prid=7744834)
```suggestion
  ["test_5fassert_5fbits_5flow_5fmessage_959",['TEST_ASSERT_BITS_LOW_MESSAGE',['../unity_8h.html#a58a1013db321e096d146bf7a90968f84',1,'unity.h']]],
```